config/locales/diaspora/en.yml

```diff
@@ -1308,6 +1308,40 @@ en:
       email_confirmed: "Email %{email} activated"
       email_not_confirmed: "Email could not be activated. Wrong link?"
 
+  two_factor_auth:
+    title: "Two factor authentication"
+    activated:
+      status: "Two factor authentication activated"
+      change_label: "Deactivate two factor authentication by entering a TOTP token."
+      change_button: "Deactivate"
+    deactivated:
+      status: "Two factor authentication not activated"
+      change_label: "Activate two factor authentication"
+      change_button: "Activate"
+    confirm:
+      title: "Confirm activation"
+      status: "Two factor authentication is not fully activated yet, you need to confirm activation with a TOTP token"
+      scan_title: "Scan the QR code"
+      scan_explanation: "Please scan the QR code with a TOTP capable app (eg FreeOTP (Android, iOS), SailOTP (SailfishOS))."
+      manual_explanation: "In case you can't scan the QR code automatically you can manually enter the secret."
+      input_title: "Confim with TOTP token"
+      input_explanation: "After scanning or entering the secret, enter the 6 digit code you see and confirm the setup."
```

Comment:
Ditto: hyphen in '6-digit'. My personal preference would be to spell it out as 'six-digit', which I think looks better.